### PR TITLE
show message when no .env file

### PR DESCRIPTION
**Pull request by chrd717**

### CODE REVIEW

File: config.ts [hunk 1]
```diff
@@ -11,4 +11,4 @@ export const faucetClient = new FaucetClient(NODE_URL, FAUCET_URL);
 export const aptosClient = new AptosClient(NODE_URL);
 
 const publisherPrivateKey = Uint8Array.from(Buffer.from(PRIVATE_KEY.replace("0x", ""), 'hex'));
-export const publisherAccount = new AptosAccount(publisherPrivateKey, PUBLIC_KEY);
\ No newline at end of file
+export const publisherAccount = !PUBLIC_KEY? undefined : new AptosAccount(publisherPrivateKey, PUBLIC_KEY);
```
Comment (sanbir) [inline config.ts:14]: cool stuff

**Review by sanbir — APPROVED** [review]
Awesome change!!

File: config.ts (hunk 1 above)
Comment (ifelsedeveloper) [reply config.ts:14]: nicely done

**Review by ifelsedeveloper — COMMENTED** [review]
looks fine

**Pull request merged**